deploy/role.yaml

```diff
@@ -11,14 +11,47 @@ rules:
   - ""
   resources:
   - pods
-  - services
-  - services/finalizers
-  - endpoints
+  verbs:
+  - get
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - create
+  - delete
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+- apiGroups:
+  - ""
+  resources:
   - persistentvolumeclaims
+  verbs:
+  - get
+  - list
+  - watch
+  - update
+- apiGroups:
+  - ""
+  resources:
   - persistentvolumes
+  verbs:
+  - get
+  - list
+  - watch
+  - update
+  - create
+  - delete
```

Comment:
why operator need to delete\create PV? its the driver that need that.
what do i miss?

Reply:
Please see my comment under the ticket first, as I said:

>  in order to create all the clusterroles of the driver, the operator's clusterrole needs all the permissions inside these clusterroles, please make sure the rules in the clusterroles of the driver is correct